src/Analyzer.Utilities/Extensions/IEnumerableOfIMethodSymbolExtensions.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities
+{
+    public static class IEnumerableOfIMethodSymbolExtensions
+    {
+        /// <summary>
+        /// Returns a list of method symbols from a given list of the method symbols, which has its parameter type as
```

Comment:
Seems like a very specific helper. Are you planning to use this for any future analyzers? Otherwise, I would recommend moving this into the analyzer file.